frontend/src/components/LocationSearchInput.tsx

```diff
@@ -25,9 +26,15 @@ export type LocationSearchInputProps = {
 };
 
 export const LocationSearchInput = (props: LocationSearchInputProps): JSX.Element => {
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
Why introduce another state here? Could you just have used state === pending?

Reply:
No, but don't know why. maybe because of the debounce?

Reply:
I dont get it, why dont just display the loading animation when the state is pending?

Reply:
I tried, but it didn't work. Feel free to try again and commit it.

Reply:
Also: the state is not pending when the debounce is still cooling down. Which is technically correct but I'm not sure that's what we want to show to the user.

Reply:
[a4bd0c6](https://github.com/Oeffis/Oeffis/pull/288/commits/a4bd0c6c7472c5c57e9929867489f03e0abbcd7b)